demo/src/components/markdown-page.js

```diff
@@ -1,25 +1,27 @@
 import React, {PureComponent, PropTypes} from 'react';
 import marked from 'marked';
 import * as Demos from './demos';
+import {markdownFiles} from '../constants/pages';
 
 /**
  * This map allows you to rewrite urls present in the markdown files
  * to be rewritted to point to other targets. It is useful so that
  * links can works both by browsing the docs using the Github explorer
  * and on the website.
  *
- * If you specify a value of false, it will transform the link to a simple
+ * If rewrite returns a falsy value, it will transform the link to a simple
  * text, in case you don't want this link to point to anything.
  */
-const urlRewrites = {
-  '/docs/layers/base-layer.md': '#/layers/catalog/layer-base-class',
-  '/docs/custom-layers.md#layerencodepickingcolorindex--number': false,
-  '/docs/layers/arc-layer.md': '#/layers/core-layers/arc-layer',
-  '/docs/layers/line-layer.md': '#/layers/core-layers/line-layer',
-  '/docs/layers/geojson-layer.md': '#/layers/core-layers/geojson-layer',
-  '/docs/layers/scatterplot-layer.md': '#/layers/core-layers/scatterplot-layer',
-  '/docs/64-bits.md': '#/documentation/advanced-topics/64-layers'
-};
+const urlRewrites = [
+  {
+    test: /^\/(docs\/.+?\.md)(#.*)?$/,
```

Comment:
oh... regex... * _ * I would always want a plain english explanation for what a regex is doing...